package.json

```diff
@@ -83,6 +83,7 @@
     "jstz": "~1.0.7",
     "lodash": "^4.17.10",
     "multiselect": "~0.9.12",
+    "number_helpers": "^0.1.1",
```

Comment:
I tried packaging this, but it fails, I assume before merge we would need a packaging ACK.

Reply:
I think that was a warning for a while already but likely that there's an update which makes it strict. I'll have a look.

Reply:
also TODO: add this package to webpack vendor file (can be done in a follow up pr).

Reply:
https://github.com/theforeman/foreman-packaging/pull/3174. Luckily in https://github.com/dLobatog/npm2rpm/pull/48 I already fixed that issue.

Reply:
@ekohl great, once you are ready please ack this PR.